kafka/build.yml

```diff
@@ -1,7 +1,12 @@
 repository: monasca/kafka
 variants:
+  - tag: 0.9.0.1-2.11-1.1.0
+    aliases:
+      - :latest
+    args:
+      KAFKA_VERSION: "0.9.0.1"
+      SCALA_VERSION: "2.11"
   - tag: 0.9.0.1-2.11-1.0.2
```

Comment:
This tag should actually be removed or it will overwrite the existing image (needed in helm for now) with the same content as the new version.